src/view/frontend/templates/category/remove_category_filter_callback.phtml

```diff
@@ -0,0 +1,20 @@
+<?php /** @var Magento\Framework\View\Element\Template $block */ ?>
+
+<script>
+    document.addEventListener('ffReady', function (ff) {
+
+        ff.resultDispatcher.addCallback("asn", function (asnData) {
+            if (factfinder.communication.Util.isNg()) {
+                asnData.splice(asnData.findIndex(element=>element.__associatedFieldNameV4 === '<?= $block->getData('category_path_field_name')?>'), 1)
```

Comment:
this needs to be changed unfortunately. Since js code is not transpilled, using arrow function will throw an error on IE11 :/